README.md

```diff
@@ -8,6 +8,41 @@ Yet another note taking tool.
 cargo tauri dev
 ```
 
+## Setup environment
+This guide focuses on how to setup your development system based on ubuntu:jammy.
+
+1. Install dependencies
+    ```bash
+    sudo apt-get update
+    sudo apt-get install -y \
+        libwebkit2gtk-4.0-dev \
+        build-essential \
+        curl \
+        wget \
+        file \
+        libssl-dev \
+        libgtk-3-dev \
+        libayatana-appindicator3-dev \
+        librsvg2-dev
+    ```
+2. Install rust
+    ```bash
+    curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+    ```
+3. Install npm (node.js)
+    ```bash
+    curl -fsSL https://deb.nodesource.com/setup_20.x | sudo -E bash - &&\
+    sudo apt-get install -y nodejs
+    ```
```

Comment:
Some goes here. Please remove the section and just state, that npm is needed. For npm there are just more installation methods than for rust.

```suggestion
```

Reply:
This section is removed with commit c337707716c614b8271430b3999e47c3e33cc656 and only states the need to have it installed beforehand.